File: engine/engine-model/src/main/scala/tools/packet/family/SendFamilyInvite.scala
package tools.packet.family

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class SendFamilyInvite(private var _characterId: Int, private var _inviter: String) extends PacketInput {
  def characterId: Int = _characterId

  def inviter: String = _inviter

  override def opcode(): SendOpcode = SendOpcode.FAMILY_JOIN_REQUEST
}

File: engine/engine-model/src/main/java/tools/packet/family/SendFamilyInvite.java
package tools.packet.family;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record SendFamilyInvite(Integer characterId, String inviter) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.FAMILY_JOIN_REQUEST;
   }
}